gems/pending/VolumeManager/MiqVolumeManager.rb

```diff
@@ -142,6 +144,24 @@ def parseLvmMetadata(pvHdrs)
     end
   end
 
+  # 1297019 remove volume groups containing logical volumes w/ 'thin-pool' segments,
+  #         as well as the volumes references in those segments' 'metadata' and 'pool' entries
+  def remove_thin_volumes
```

Comment:
@movitto 
This method removes volume groups that contain thin pool logical volumes. However, I haven't seen anything that indicates that a volume group can't contain both, regular and thin logical volumes.

If that's the case, we can't remove the volume groups, we probably should filter the logical volumes out of the `@logicalVolumes` array in the `initialize` method of `MiqVolumeManager`. In addition, while this code addresses **thin pool**, **tmeta** and **tdata** volumes, it needs to identify **thin volumes** - the volumes that reference **thin pool** volumes.